scripts/dataset/export/index.test.js

```diff
@@ -0,0 +1,148 @@
+    const givenContentPath = this._obj;
+
+    const result = dircompare.compareSync(givenContentPath, expectedContentPath, {
+      excludeFilter: '.DS_Store',
+      compareContent: true,
+    });
+
+    this.assert(
+      result.same,
+      generateFailureMessage(result),
+      `expected ${givenContentPath} to have a different content as ${expectedContentPath}`,
+    );
+
+    function generateFailureMessage(result) {
+      let message = `expected ${givenContentPath} to have the same content as ${expectedContentPath}
+
+     Statistics - equal entries: ${result.equal}, distinct entries: ${result.distinct}, left only entries: ${result.left}, right only entries: ${result.right}, differences: ${result.differences}\n`;
+
+      result.diffSet.forEach(diff => {
+        if (diff.state == 'equal') {
+          return;
+        }
+
+        message += `     Reason: ${diff.reason}: File - name1: ${diff.name1}, type1: ${diff.type1}, name2: ${diff.name2}, type2: ${diff.type2}\n`;
+      });
+
+      return message;
+    }
+  });
+});
+
+describe('Export', () => {
+  describe('#generateArchive', () => {
+    let storageAdapter;
+    const archiveName = 'test-dataset';
+    const archivePath = path.resolve(__dirname, `./tmp/${archiveName}.zip`);
+    const tmpPath = path.resolve(__dirname, './tmp');
+    const datasetFixturesPath = path.resolve(__dirname, './test/fixtures/dataset');
+    let zip;
+
+    before(async function () {
+      this.timeout(10000);
+      storageAdapter = new GitAdapter({
+        ...config.get('recorder.versions.storage.git'),
+        path: path.resolve(__dirname, '../../../', config.get('recorder.versions.storage.git.path')),
+        fileExtension: 'md',
+      });
+
+      await storageAdapter.initialize();
+
+      await storageAdapter.record({
+        serviceId: FIRST_SERVICE_PROVIDER_ID,
+        documentType: FIRST_DOCUMENT_TYPE,
+        content: FIRST_CONTENT,
+        fetchDate: FIRST_FETCH_DATE,
+        snapshotId: SNAPSHOT_ID,
+      });
+
+      await storageAdapter.record({
+        serviceId: FIRST_SERVICE_PROVIDER_ID,
+        documentType: FIRST_DOCUMENT_TYPE,
+        content: SECOND_CONTENT,
+        fetchDate: SECOND_FETCH_DATE,
+        snapshotId: SNAPSHOT_ID,
+      });
+
+      await storageAdapter.record({
+        serviceId: SECOND_SERVICE_PROVIDER_ID,
+        documentType: FIRST_DOCUMENT_TYPE,
+        content: FIRST_CONTENT,
+        fetchDate: THIRD_FETCH_DATE,
+        snapshotId: SNAPSHOT_ID,
+      });
+
+      await storageAdapter.record({
+        serviceId: SECOND_SERVICE_PROVIDER_ID,
+        documentType: SECOND_DOCUMENT_TYPE,
+        content: SECOND_CONTENT,
+        fetchDate: FOURTH_FETCH_DATE,
+        snapshotId: SNAPSHOT_ID,
+      });
+
+      await generateArchive({
+        archivePath,
+        releaseDate: new Date(RELEASE_DATE),
+      });
+
+      zip = new StreamZip.async({ file: archivePath }); // eslint-disable-line new-cap
```

Comment:
I would prefer that we use the [`properties: false`](https://eslint.org/docs/rules/new-cap#properties) option of the `new-cap` ESlint rule instead of a `disable-line` statement 🙂